chemprop/nn/predictors.py

```diff
@@ -203,6 +204,30 @@ def forward(self, Z: Tensor) -> Tensor:
     train_step = forward
 
 
+@PredictorRegistry.register("regression-quantile")
+class QuantileFFN(RegressionFFN):
+    n_targets = 2
+    _T_default_criterion = QuantileLoss
+
+    def forward(self, Z: Tensor) -> Tensor:
+        Y = super().forward(Z)
+        lower_pred, upper_pred = torch.chunk(Y, self.n_targets, 1)
+
+        lower_pred = self.output_transform(lower_pred)
+        upper_pred = self.output_transform(upper_pred)
+
+        mean = (lower_pred + upper_pred) / 2
+        interval = upper_pred - lower_pred
+
+        return torch.cat((mean, interval), 1)
+
+    def train_step(self, Z: Tensor) -> Tensor:
+        Y = super().forward(Z)
+        mean, interval = torch.chunk(Y, self.n_targets, 1)
```

Comment:
isn't the output from `super().forward()` a tensor containing the lower and upper bound of prediction? At least that's what L214 implies.

Reply:
Yes, it is a tensor containing both lower and upper bounds.

Reply:
so this should be
```suggestion
        lower, upper = torch.chunk(Y, self.n_targets, 1)
```
then, right?

Reply:
Oops! I intended to use `Y = self.forward(Z)` instead of `Y = super().forward(Z)` here. Thus, the output is mean and interval, which are further used to reconstruct the upper and lower bounds in `QuantileLoss` to calculate the loss.